### PR TITLE
v1 of react hooks and trip modal component 

### PR DESCRIPTION
**Pull request by kenlitton**
hopefully this works

### CODE REVIEW

File: client/components/NewTripModal.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+
+Modal.setAppElement('#root');
```
Comment (lawrencehan650) [inline client/components/NewTripModal.jsx:4]: When testing this out, we were able to comment it out and functionality remained the same. We should investigate further.

File: index.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <!-- <link rel="stylesheet" href=""> -->
   </head>
   <body>
-    <h1>App</h1>
+    <h1 id='head'>App</h1>
```
Comment (lawrencehan650) [inline index.html:9]: Delete` id='head'`

**Review by lawrencehan650 — CHANGES_REQUESTED** [review]
Hooks look great!

File: index.html (hunk 2 above)
Comment (kenlitton) [reply index.html]: agreed

**Review by kenlitton — COMMENTED** [review]
updated the header tag in index.html

**Review by cssim22 — APPROVED** [review]
Agree to changes requested

**Review by lawrencehan650 — APPROVED** [review]

**Pull request merged**